evi check/Form1.cs

```diff
@@ -0,0 +1,650 @@
+                        {
+                            this.Evi1EMF5.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2EMF5.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3EMF5.Checked = false;
+                        }
+                        break;
+
+                    case "GhostWriting":
+                        if (group == 1)
+                        {
+                            this.Evi1GhostWriting.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2GhostWriting.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3GhostWriting.Checked = false;
+                        }
+                        break;
+
+                    case "FingerPrints":
+                        if (group == 1)
+                        {
+                            this.Evi1FingerPrints.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2FingerPrints.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3FingerPrints.Checked = false;
+                        }
+                        break;
+
+                    case "SpiritBox":
+                        if (group == 1)
+                        {
+                            this.Evi1SpiritBox.Checked = false;
+                        }
+                        else if (group == 2)
+                        {
+                            this.Evi2SpiritBox.Checked = false;
+                        }
+                        else if (group == 3)
+                        {
+                            this.Evi3SpiritBox.Checked = false;
+                        }
+                       // this.Evi1SpiritBox.Checked = false;
+                        break;
+                }
+            }
+
+        }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        private void Evi1Cold_click(object sender, EventArgs e)
+        {
+
+            if (this.Evi1Cold.Checked == true)
+            {
+                evi1 = null;
+                Resetevibutton("", 1);
+            }
+            else
+            {
+                evi1 = "Cold";
+                Resetevibutton("Cold", 1);
+                this.Evi1Cold.Checked = true;
+                // Show_output();
+            }
+            Show_output();
+        }
+      
+        private void Evi1GhostOrb_click(object sender, EventArgs e)
+        {
+            if (this.Evi1GhostOrb.Checked == false)
```

Comment:
The `== false` here is redundant. Prefer the `!` operator
```suggestion
            if (!this.Evi1GhostOrb.Checked)
```